cluster/manifests/logging-agent/config.yaml

```diff
@@ -42,6 +55,18 @@ data:
       @type token
     </filter>
 
+    <filter scalyr-agent.**>
+      @type record_transformer
+      <record>
+        # s3_path is: ${application}/${component}/${namespace}/${environment}/${version}/${cont_name}
+        s3_path     "logging-agent/logging/visibility/{{ .Environment }}/v0.30/scalyr-agent"
+        application "logging-agent"
+        version     "v0.30"
```

Comment:
Is it possible to extract version from `deamonset.yaml`, or declare it in another place and use in this two?

Reply:
We also hardcode it here https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/cluster/manifests/logging-agent/daemonset.yaml#L9
I do not know. Can we introduce some kind of variable in another manifest and reuse it here?
Do you know @mikkeloscar ?

Reply:
Hardcode in one place is declaration. On several places — unmaintainable thing

Reply:
I can't think of any way to do this now.

Reply:
I mean you could set the variable in `cluster/config-defaults.yaml` and use that, but I don't know how obvious it is when you need to change it.

Reply:
You could also move both resources into one yaml and then use template variables to define it once. Not sure if you think it'll be uglier or not. :)